Source/LibationAvalonia/Program.cs

```diff
@@ -30,6 +30,14 @@ static async Task Main()
 			var classicLifetimeTask = Task.Run(() => new ClassicDesktopStyleApplicationLifetime());
 			var appBuilderTask = Task.Run(BuildAvaloniaApp);
 
+			if (App.IsWindows)
```

Comment:
Ah, I misunderstand that one comment of yours regarding this. I like this. Much thanks.

Reply:
To be clear, this is one of the 2 ways that version/platform are tracked.  The GUI apps set ReleaseIdentifier, which is logged in LibationScaffolding.logStartupState().

In addition, the new preprocessor definitions in AppScaffolding.csproj are also used by LibationScaffolding.logStartupState() to log the OS.